### ISSUE TITLE
Google Chat Notification is waiting for a checkpoint

### ISSUE THREAD

**Issue by prawen** [jenkinsci/google-chat-notification-plugin#8]
Hello,

I'm seeing this issue, when there are merge conflicts. Its taking 12 minutes. Is there any way to define the timeout value?

```
00:04:48.755 ERROR: Branch not suitable for integration as it does not merge cleanly: Command "git merge -s resolve --ff de2dd5817d98264a712a62ce6eb98a4f134851bc" returned status code 1:
00:04:48.755 stdout: Trying really trivial in-index merge...
00:04:48.755 Nope.
00:04:48.755 Trying simple merge.
00:04:48.755 Simple merge failed, trying Automatic merge.
00:04:48.755 Auto-merging abcd
00:04:48.755 Auto-merging dabcd
00:04:48.755 Auto-merging abcd
00:04:48.755 Auto-merging abcd
00:04:48.755 Auto-merging abcd
00:04:48.755 Auto-merging abcd
00:04:48.755 Auto-merging abcd

00:04:48.755 Automatic merge failed; fix conflicts and then commit the result.
00:04:48.755 
00:04:48.755 stderr: error: Merge requires file-level merging
00:04:48.755 ERROR: abcd
00:04:48.755 ERROR: content conflict in abcd

00:04:48.755 fatal: merge program failed
00:04:48.755 
00:04:48.755 [CHECKSTYLE] Skipping publisher since build result is FAILURE
00:04:48.755 Google Chat Notification is waiting for a checkpoint on build #476
00:17:58.377 ERROR: Step ‘Google Chat Notification’ aborted due to exception: 
00:17:58.378 java.lang.InterruptedException
00:17:58.378 	at java.lang.Object.wait(Native Method)
00:17:58.378 	at java.lang.Object.wait(Object.java:502)
00:17:58.378 	at hudson.model.Run$RunExecution$CheckpointSet.waitForCheckPoint(Run.java:1681)
00:17:58.378 	at hudson.model.Run.waitForCheckpoint(Run.java:1634)
00:17:58.378 	at hudson.model.CheckPoint.block(CheckPoint.java:157)
00:17:58.378 	at hudson.tasks.BuildStepMonitor$3.perform(BuildStepMonitor.java:41)
00:17:58.378 	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:744)
00:17:58.378 	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:690)
00:17:58.378 	at hudson.model.Build$BuildExecution.post2(Build.java:186)
00:17:58.378 	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:635)
00:17:58.378 	at hudson.model.Run.execute(Run.java:1840)
00:17:58.378 	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
00:17:58.378 	at hudson.model.ResourceController.execute(ResourceController.java:97)
00:17:58.378 	at hudson.model.Executor.run(Executor.java:429)
```

**Comment by prawen** [comment]
That would be great.

**Comment by chiragnaik** [comment]
This is released in latest 1.3 release

**Comment by VolvoxGlobator** [comment]
Sorry for little bit of necro, but I am still running into this issue on version 1.4, Jenkins 2.319.3
